src/main.js

```diff
@@ -50,6 +66,23 @@ program.action(async (files) => {
   }
 });
 
+function readConfigFile() {
```

Comment:
PRAISE! LGTM! The function works as intended.

Suggestion:
I would suggest having the function in its own file to keep the consistency of the structure. You can create a folder called `helpers/` or `utils/` and export it from there!

Reply:
I moved it to the utils folder.